lib/amqp/client/channel.rb

```diff
@@ -173,6 +173,23 @@ def basic_cancel(consumer_tag, no_wait: false)
       consumer.close
     end
 
+    def basic_ack(delivery_tag, multiple: false)
+      write_bytes FrameBytes.basic_ack(@id, delivery_tag, multiple)
+    end
+
+    def basic_nack(delivery_tag, multiple: false, requeue: false)
+      write_bytes FrameBytes.basic_nack(@id, delivery_tag, multiple, requeue)
+    end
+
+    def basic_reject(delivery_tag, requeue: false)
+      write_bytes FrameBytes.basic_reject(@id, delivery_tag, requeue)
+    end
+
+    def basic_qos(prefetch_size, prefetch_count, global: false)
```

Comment:
Prefetch size is like never used, i don't know if it's even supported in rabbitmq. Let's make that one an optional argument.

Reply:
3355079a2a9018b34bfbeb47792348019235ab87